SAKPaaS/src/app/core/services/badge.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class BadgeService {
+
+  constructor() { }
+}
+
+export enum BadgeType {
```

Comment:
Enums sollten nicht in einer `.service.ts` file exportiert werden.

Reply:
@timgrohmann Du hattest das so gemacht🤔 Kann das in badge.interface.ts?

Reply:
Also unter `SAKPaaS\src\app\core\enums` einfach eine `badge-type.enum.ts` anlegen @luisafuss :)

Reply:
böser Tim :D

Reply:
Jaa ich weiß doch xD Kann überall anders hin, da war es halt am schnellsten reingehauen ;)